spec/models/match_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe Match do
+  let(:match) do
+    build(:match, points_for_team1: 8,
+                  points_for_team2: 2)
+  end
+  let(:user) { create(:user) }
+  context "is valid" do
```

Comment:
"is valid when" so when the test output text appears it says "is valid when it .."